app/src/main/java/org/oppia/app/options/OptionsFragmentPresenter.kt

```diff
@@ -133,30 +155,78 @@ class OptionsFragmentPresenter @Inject constructor(
         profileManagementController.updateReadingTextSize(
           profileId,
           ReadingTextSize.SMALL_TEXT_SIZE
+        ).observe(
+          fragment,
+          Observer {
+            if (it.isSuccess()) {
+              readingTextSize = ReadingTextSize.SMALL_TEXT_SIZE
+            } else {
+              consoleLogger.e(
+                READING_TEXT_SIZE_TAG,
+                "${READING_TEXT_SIZE_ERROR}: small text size",
```

Comment:
You can write this directly like this:
`$READING_TEXT_SIZE_ERROR: small text size`

Here and everywhere else.

Once done, you can resolve this comment on your own.